packages/resource-deployment/scripts/run-command-on-all-vmss-for-pool.sh

```diff
@@ -76,67 +80,54 @@ getVmssInfo() {
 
 setupVmss() {
     for vmssResourceGroup in "${vmssResourceGroups[@]}"; do
-        echo "Enabling system-assigned managed identity for VMSS resource group $vmssResourceGroup"
+        echo "Running setup for VMSS resource group $vmssResourceGroup for pool $pool"
```

Comment:
the script comment needs to be updated `# The script will enable system-assigned managed identity on Batch pool VMSS`